js/searchbar/customBangs.js

```diff
@@ -12,6 +12,7 @@ var urlParser = require('util/urlParser.js')
 var { db } = require('util/database.js')
 var formatRelativeDate = require('util/relativeDate.js')
 var contentBlockingToggle = require('navbar/contentBlockingToggle.js')
+const settings_ = require('util/settings/settings.js');
```

Comment:
This can probably be `var settings = `

Reply:
Tried that, settings was already defined.

Reply:
Actually, we don't need that loaded in.